mobility_games/games/wall_audio_player.py

```diff
@@ -91,8 +104,13 @@ def run(self):
                         #else:
                         freq = ac.quantize(freq, quantizetype = self.quantizetype, key=self.key)
                         #print (freq)
+                    else:
+                        if (self.pitchdefault != ''):
+                            freq = ac.freq(self.pitchdefault)
                     if (self.vol):
                         vol = (.8/(1+math.exp((self.dist)/1.5-1))+.2)*vol#insert volume function here. that goes from 1 to 0 with increasing distance to wall.
+                    else:
+                        vol = self.voldefault
                     #self.synth = ac.delay(self.synth, 1, self.s*.2, .5) #add delay to the synth
 
                     if self.track:
```

Comment:
Isn't this...
`if self.track:`
`    pass`
`else:`

...functionally identical to:
`if not self.track:`

Is there a reason why you have it as an if/else?